frontend/unpack-pro-icons.sh

```diff
@@ -1,17 +0,0 @@
-#!/bin/sh -eu
-
-# SPDX-FileCopyrightText: 2017-2022 City of Espoo
-#
-# SPDX-License-Identifier: LGPL-2.1-or-later
-
-VERSION=6.4.0
-
-unpack() {
-  NAME=${1}
-  mkdir -p node_modules/@fortawesome/"${NAME}"
-  tar xz --strip-components 1 -C node_modules/@fortawesome/"${NAME}" -f vendor/fortawesome/fortawesome-"${NAME}"-"${VERSION}".tgz
-}
-
-unpack pro-light-svg-icons
-unpack pro-regular-svg-icons
-unpack pro-solid-svg-icons
```

Comment:
This script needs to be kept for backwards compatibility, until other cities change their way of installing the pro icons.

Reply:
renamed to setup-pro-icons.sh and brought back init/unpack scripts.